brianlong/www.validators.app
# 186743657 - Update ping thing sender filter

#### What's this PR do?
- Modifies posted_by filter in PingThings in order to use a wildcard % at the end of string.

#### How should this be manually tested?
- Create users in your db with similar usernames, eg.:
  -  "alfreda.walsh"
  -  "alfreda.walsh.senior"
  -  "junior.alfreda.walsh"
-  Populate Ping Things table: `rails r dev/create_test_ping_things.rb` (this script will create pings for all users)

- Enter the ping thing page and use the posted_by filter. Check the following scenarios:
  -  enter only the username, eg. "alfreda.walsh". The result should return "alfreda.walsh" and "alfreda.walsh.senior" records
  - use "alfreda.walsh.senior" and output should include "alfreda.walsh.senior" only
  - "junior.alfreda.walsh" should return only the "junior.alfreda.walsh" records

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186743657)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
